zcash_proofs/src/sapling/prover.rs

```diff
@@ -261,7 +262,7 @@ impl SaplingProvingContext {
         let bvk = PublicKey::from_private(&bsk, FixedGenerators::ValueCommitmentRandomness, params);
 
         // In order to check internal consistency, let's use the accumulated value
-        // commitments (as the verifier would) and apply valuebalance to compare
+        // commitments (as the verifier would) and apply value balance to compare
```

Comment:
This is referring top a specific field name:
```suggestion
        // commitments (as the verifier would) and apply value_balance to compare
```